crates/typst-library/src/math/mod.rs

```diff
@@ -324,7 +324,7 @@ impl LocalName for EquationElem {
             Lang::ARABIC => "معادلة",
             Lang::BOKMÅL => "Ligning",
             Lang::CHINESE if option_eq(region, "TW") => "方程式",
-            Lang::CHINESE => "等式",
+            Lang::CHINESE => "公式",
```

Comment:
could you explain what the previous thing meant and what the change does? the original name is from this PR: https://github.com/typst/typst/pull/481 @1bitbool do you have any opinion on this change?

Reply:
In Mainland China, we don't use '等式' in document for 'equation' but use '公式' instead, because some history reason.

You can check this standard [`GB/T 7713.1-2006`](https://github.com/Haixing-Hu/typesetting-standard/blob/master/%E5%9B%BE%E4%B9%A6%E3%80%81%E6%9C%9F%E5%88%8A%E3%80%81%E8%AE%BA%E6%96%87%E7%9A%84%E7%BC%96%E6%8E%92/%E3%80%90GB%3AT%207713.1-2006%E3%80%91%E5%AD%A6%E4%BD%8D%E8%AE%BA%E6%96%87%E7%BC%96%E5%86%99%E8%A7%84%E5%88%99.pdf), The Rules of Dissertation Writing, Sec. 5.3.4, it uses '公式':
![image](https://github.com/typst/typst/assets/358450/23533ac5-a3d5-4b45-b481-2e02737b94db)

Reply:
I have looked up some references, and "equation" directly corresponds to "等式" or "方程". "公式" should be translated as "formula". However, as @kingfree mentioned, due to some historical reasons, "公式" has taken on a broader meaning in academic papers, also encompassing the meaning of "equation".